lib/reek/code_comment.rb

```diff
@@ -94,16 +95,20 @@ def initialize(detector_name:, original_comment:, line:, source:, options: {})
         @line             = line
         @source           = source
         @options          = options
+        @detector_class   = nil # We only know this one after our first initial checks
+        @parsed_options   = nil # We only know this one after our first initial checks
```

Comment:
I think this is telling us this should be restructured, but I'm not sure how yet.